lib/resources/auth_manager.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+/*
+  AuthManager class manages user authentication token.
+ */
+class AuthManager {
```

Comment:
Does it make sense to rename this class to AuthRepository. Since that's the convention we would like to follow for accessing a resource, be it from network or local storage.

Reply:
Changed the file name.